napari/layers/vectors/vectors.py

```diff
@@ -717,7 +717,7 @@ def _update_slice_response(self, response: _VectorSliceResponse):
     def _update_thumbnail(self):
         """Update thumbnail with current vectors and colors."""
         # Set the default thumbnail to black, opacity 1
-        colormapped = np.zeros(self._thumbnail_shape)
```

Comment:
Is this (and the one below) a functional change?

Reply:
This is because the thumbnail needs to have the specific dtype `uint8`.